Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -659,8 +668,8 @@ public void MigrateToPlanner(PlotMode plotMode) {
         }
     }
 
-    public void MigrateToInstance(int mapId, long ownerId) {
-        bool instancedContent = ServerTableMetadata.InstanceFieldTable.Entries.ContainsKey(mapId);
+    public void Migrate(int mapId, long ownerId = 0) {
+        bool isInstanced = ServerTableMetadata.InstanceFieldTable.Entries.ContainsKey(mapId);
```

Comment:
I wonder if we can remove this and just have the check happen on world server instead. just a thought. doesn't have to happen

Reply:
we need to save the return channel, so i think it's valid to stay in game server